lockfile_unix.go

```diff
@@ -194,7 +194,7 @@ func (l *lockfile) Modified() (bool, error) {
 		return true, err
 	}
 	if n != len(id) {
-		return true, unix.ENOSPC
```

Comment:
Was this change in for the tests?  If not, could you add a little blurb in the PR description for hysterical purposes?

Reply:
I'd forgotten that we were returning an error for cases where the lock file is newly-created, which on retrospect didn't make a lot of sense, since we don't enforce a size for the last-writer identifier.